### PR TITLE
Issue-19: embeds must have a line break before and after the url

### PR DESCRIPTION
**Pull request by mogmarsh**
**Issue Number:** #19

**Issue URL:** [embeds must have a line break before and after the url](https://github.com/alleyinteractive/wp-block-converter/issues/19)

### Description of the Change

- Fixed an issue where embeds do not display correctly on the front end if they do not have a line break before and after the URL.
- Implemented preservation of new lines in embed blocks to ensure accurate front-end display.

### How to Verify

1. Convert an embed and view it on the front end.
2. Edit the block to add line breaks before and after the URL.
3. Verify that the converted embed retains line breaks and displays correctly.

### Additional Notes

_No additional notes._

### CODE REVIEW

File: src/class-block-converter.php [hunk 1]
```diff
@@ -446,8 +447,13 @@ public static function get_node_tag_from_html( $html, $tag = 'body' ) {
 	 * @return string
 	 */
 	protected function minify_block( $block ) {
-		if ( preg_match( '/(\s){2,}/s', $block ) === 1 ) {
-			return preg_replace( '/(\s){2,}/s', '', $block );
+		if ( \str_contains( $block, 'wp-block-embed' ) ) {
```
Comment (attackant) [inline src/class-block-converter.php:450]: Are there any other cases where we want to keep newlines?

**Review by attackant — APPROVED** [review]
Question for you, no blockers 🐙

File: src/class-block-converter.php (hunk 1 above)
Comment (mogmarsh) [reply src/class-block-converter.php:450]: @attackant Not that I know of. The embed support is a few weeks old (I added it), but the rest of it has been working fine for at over two years, so I'm hesitant to change anything else.

**Review by juliobranha — COMMENTED** [review]
🐉

**Pull request merged**